3scale/porta
# THREESCALE-1789: Add "Managed by operator" banner (UI)

**What this PR does / why we need it**:

The operator can manage porta resources through API. When a resource is managed by operator, we must inform the user about that.

https://github.com/3scale/porta/pull/3857 implements the backend support (annotations). This PR implements the frontend, for now, just a banner in the relevant screens:

- `http://provider-admin.3scale.localhost:3000/buyers/accounts/4`
- `http://provider-admin.3scale.localhost:3000/buyers/accounts/4/edit`
- `http://provider-admin.3scale.localhost:3000/apiconfig/services/2`
- `http://provider-admin.3scale.localhost:3000/apiconfig/services/2/edit`
- `http://provider-admin.3scale.localhost:3000/p/admin/backend_apis/2`
- `http://provider-admin.3scale.localhost:3000/p/admin/backend_apis/2/edit`

Examples:

![image](https://github.com/user-attachments/assets/42da796a-67c1-4b00-b560-699cca816beb)

![image](https://github.com/user-attachments/assets/cb661f29-6438-4eb8-93b2-87cc2adc3c7a)

**Which issue(s) this PR fixes** 

https://issues.redhat.com/browse/THREESCALE-1786

**Verification steps** 

1. Marks a resource as managed by operator. Check https://github.com/3scale/porta/pull/3857 to know how
2. Visit the resource `#show` or `#edit` screens
3. You should see the banner
